l1-contracts/contracts/governance/ChainAdmin.sol

```diff
@@ -38,6 +69,8 @@ contract ChainAdmin is IChainAdmin, Ownable2Step {
         require(_calls.length > 0, "No calls provided");
         // solhint-disable-next-line gas-length-in-loops
         for (uint256 i = 0; i < _calls.length; ++i) {
```

Comment:
Also I want to add ReentrancyGuard for `multicall` to avoid side effect issues with different restriction modules.